### ISSUE TITLE
Input field hint text accessibility and component improvement

### ISSUE THREAD

**Issue by enowak1031** [trimble-oss/modus-web-components#2517]
Input field formatting change based on recommendation/ submission for better accessibility: https://trimble-arch.aha.io/ideas/ideas/MOD-I-70

Waiting on Figma updates: https://jira.trimble.tools/browse/MODUS-49

Related Issue: #2281

**Comment by enowak1031** [comment]
<img width="764" alt="Screenshot 2024-05-17 at 11 28 30 AM" src="https://github.com/trimble-oss/modus-web-components/assets/97305117/85c16a1b-bf77-4d42-8ba9-40adfa6bfa93">

**Comment by cjwinsor** [comment]
1 is already complete
2 is in progress with #2565 
3 is not started
4 done but missing the icons
* Also no icon for success message
